Bigtable/src/DataClient.php

```diff
@@ -436,4 +437,96 @@ public function sampleRowKeys(array $options = [])
             ];
         }
     }
+
+    /**
+     * Mutates the specified row atomically based on output of the filter.
+     *
+     * Example:
+     * ```
+     * use Google\Cloud\Bigtable\Mutations;
+     *
+     * $mutations = (new Mutations)->upsert('family', 'qualifier', 'value');
+     * $result = $dataClient->checkAndMutateRow('rk1', ['trueMutations' => $mutations]);
+     * ```
+     *
+     * //With predicate filter
```

Comment:
As noted by @jdpedrie this should be within the backticks. That's my mistake, as I think I introduced this form to the branch awhile back :).

Reply:
done